docs/dev/Creating UI5 Web Components Packages.md

```diff
@@ -107,42 +107,42 @@ That's it!
 
 ## Understanding the project structure
 
-### package.json
+### Package.json
 
 The initialization script will create several NPM scripts for you in `package.json`.
 
 Task | Purpose
 -----|-------
-clean | Deletes the `dist/` directory with the build output
-build | Production build to the `dist/` directory
```

Comment:
All of these are actions, so in this case Production build is the only definition that stands out. I'd replace it with the appropriate verb.